src/wp-includes/class-wp-block-bindings-registry.php

```diff
@@ -305,4 +291,24 @@ public static function get_instance() {
 
 		return self::$instance;
 	}
+
+	/**
+	 * Adds the `uses_context` to the editor settings.
+	 *
+	 * This allows to reuse the `uses_context` definition in the editor.
+	 *
+	 * @since 6.6.0
+	 *
+	 * @param array $settings The block editor settings from the `block_editor_settings_all` filter.
+	 * @return array The editor settings with extended block bindings `uses_context`.
+	 */
+	public static function add_uses_context_to_editor_settings( $settings ) {
+		$registered_block_bindings_sources = get_all_registered_block_bindings_sources();
+		foreach ( $registered_block_bindings_sources as $source ) {
+			if ( ! empty( $source->uses_context ) ) {
+				$settings['__experimentalBlockBindings'][ $source->name ]['usesContext'] = $source->uses_context;
```

Comment:
In PHP you need to initialize all the arrays at any level.

Should we also pass the `label` to the client?

Reply:
I changed the way arrays are initialized and I added the `label` as well [here](https://github.com/WordPress/wordpress-develop/pull/6456/files#diff-edfc853e670c7bfc5a838dc74f75ba5d8b95cb4adbddf0644d9227bcb692e344R658-R660).